src/service/waveHandler.js

```diff
@@ -4,20 +4,15 @@ const path = require('path')
 
 const WAVE_SUBTYPE = {}
 
-const isEmpty = obj => {
-  for (let i in obj) return false
-  return true
-}
-
 class WaveHandler extends StandardHandler {
   constructor(line, type, work, metadata) {
     super(line, type, work, metadata)
-    if (isEmpty(WAVE_SUBTYPE)) {
-      StandardHandler.metadata[this.type].content.reduce((acc, val) => {
-        acc[val.suffix] = val.xmlName
-        return acc
-      }, WAVE_SUBTYPE)
-    }
+    console.log(this.type)
```

Comment:
**question:** Is this a remaining of a debugging session or is it useful to always have this log?

Reply:
That is something I forgot to clean after the debugging session, nice catch !
Thanks